ui/fakes/fake_ui.go

```diff
@@ -93,24 +94,24 @@ func (ui *FakeUI) PrintTable(table Table) {
 	ui.Tables = append(ui.Tables, table)
 }
 
-func (ui *FakeUI) AskForText(label string) (string, error) {
+func (ui *FakeUI) AskForText(opts ui.TextOpts) (string, error) {
```

Comment:
Rename `(ui *FakeUI)` to `(fakeUI *FakeUI)` as we are using `(opts ui.TextOpts)`. It becomes difficult to understand which ui is being referred. Same for others

Reply:
made the change to import text opts with `type.TextOpts`